firebase.json

```diff
@@ -87,7 +87,7 @@
       { "source": "/devices", "destination": "/overview#platform", "type": 301 },
       { "source": "/dev{,/**}", "destination": "https://api.dart.dev/dev", "type": 301 },
       { "source": "/diagnostics", "destination": "/tools/diagnostic-messages", "type": 301 },
-      { "source": "/diagnostics/:code*", "destination": "/tools/diagnostic-messages#:code", "type": 301 },
+      { "source": "/diagnostics/:code*", "destination": "/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode#:code", "type": 301 },
```

Comment:
Should diagcode -> code? Or will have the `?` before the `#` mean that we get the `code` data?

Reply:
Fixed it! Using `utm_content` to actually log the code